docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,83 @@
+*******************************************************
+Home Directory Storage Quota
+*******************************************************
+There are strict quotas for each home directory (i.e.,  ``/home/<username>``) and staying quota is vital in preventing issues using Open OnDemand and performing tasks on the HPC. For more information about the quota see :ref:`discovery_storage`. This page provides some best practices for staying under quota.
+
+Analyze disk usage
```

Comment:
This is a section header, so capitalize each word (subsections only capitalize the first word).